main.py

```diff
@@ -66,3 +69,68 @@ async def handle_events(self, event: KytosEvent):
                 TOPIC_NAME, event.name, event.name, event.content
             )
         )
+
+    @rest("v1/create", methods=['POST'])
+    async def create_filter(self, request: Request) -> JSONResponse:
+        """
+        Tests a potential filter and adds it to the rule dictionary
+
+        Requires 1 provided value in the content section called "pattern". Accepts 1 optional
+        value called "description"
+        """
+        log.info("Request received at /v1/create")
+
+        try:
+            content = await aget_json_or_400(request)
+            pattern = content.get("pattern")
+            description = content.get("description", "N/A")
+            self._rule_ops.create_filter(pattern, description)
+
+            return JSONResponse(
+                {
+                    "status": "200",
```

Comment:
In success case we avoid having a redundant status code info in the response, similar to what we have in other NApps https://github.com/kytos-ng/mef_eline/blob/master/main.py#L217. Also, "response" key is a bit redundant in the action response.